base/mpfr.jl

```diff
@@ -283,16 +340,16 @@ end
 _cpynansgn(x::AbstractFloat, y::BigFloat) = isnan(x) && signbit(x) != signbit(y) ? -x : x
 
 Float64(x::BigFloat) =
-    _cpynansgn(ccall((:mpfr_get_d,:libmpfr), Float64, (Ref{BigFloat}, Int32), x, ROUNDING_MODE[]), x)
+    _cpynansgn(ccall((:mpfr_get_d,:libmpfr), Float64, (Ref{BigFloat}, MPFRRoundingMode), x, ROUNDING_MODE[]), x)
 Float32(x::BigFloat) =
-    _cpynansgn(ccall((:mpfr_get_flt,:libmpfr), Float32, (Ref{BigFloat}, Int32), x, ROUNDING_MODE[]), x)
+    _cpynansgn(ccall((:mpfr_get_flt,:libmpfr), Float32, (Ref{BigFloat}, MPFRRoundingMode), x, ROUNDING_MODE[]), x)
 # TODO: avoid double rounding
 Float16(x::BigFloat) = Float16(Float32(x))
 
-Float64(x::BigFloat, r::RoundingMode) =
-    _cpynansgn(ccall((:mpfr_get_d,:libmpfr), Float64, (Ref{BigFloat}, Int32), x, to_mpfr(r)), x)
-Float32(x::BigFloat, r::RoundingMode) =
-    _cpynansgn(ccall((:mpfr_get_flt,:libmpfr), Float32, (Ref{BigFloat}, Int32), x, to_mpfr(r)), x)
+Float64(x::BigFloat, r::MPFRRoundingMode) =
```

Comment:
Will `Float64(x::BigFloat, r::RoundingMode)` continue to work?

Reply:
Good catch, thanks!